stripe/src/main/java/com/stripe/android/paymentsheet/BasePaymentMethodsListFragment.kt

```diff
@@ -56,7 +56,9 @@ internal abstract class BasePaymentMethodsListFragment(
             canClickSelectedItem,
             paymentOptionSelectedListener = ::onPaymentOptionSelected,
             addCardClickListener = {
-                transitionToAddPaymentMethod()
+                if (sheetViewModel.processing.value == false) {
+                    transitionToAddPaymentMethod()
+                }
```

Comment:
Can we go one step further and observe `sheetViewModel.processing` to update the adapter to toggle `isClickable` on the item views? That would handle both this case and the need for `interactionEnabled`.

Reply:
Done